samples/system-test/subscriptions.test.js

```diff
@@ -146,6 +146,17 @@ describe('subscriptions', () => {
     const output = await execPromise(
       `${commandFor('synchronousPull')} ${projectId} ${subscriptionNameOne}`
     );
+    assert.match(output, /Hello/);
+    assert.match(output, /Done./);
+  });
+
+  it('should listen for messages synchronously with lease management', async () => {
+    pubsub.topic(topicNameOne).publish(Buffer.from(`Hello, world!`));
+    const output = await execPromise(
```

Comment:
So I know it's not part of this PR, but the `execPromise` thing happening here is silly (I think).  Is there a reason for this to be async?  The tests all run serially in their own container.

Reply:
Hmm. Yeah, it seems like execSync would be okay, but I'm going to guess that this is letting other test running code capture and process outputs while it's running. I'm not sure if that matters. I can try switching it to execSync and see if it causes any issues.

Reply:
`spawnSync` I believe gives more information about the process that ran, such as exit code; might be a good option.

Reply:
My hunch above seems to have been correct, that it was execPromise() so that other events could fire, but I also ended up thinking that that was the wrong way to handle the situation. I added an await to the stuff before it and switched to execSync(), and it works fine.